index.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
 
     let features;
     try {
-      features = this.project.require('@ember-data/private-build-infra/src/features');
+      features = this.project.require('@ember-data/private-build-infra/src/features')();
```

Comment:
Ya, FWIW, this changed in the most recent set of canaries.

I'm concerned that our `try`/`catch` didn't properly absorb this though (and fallback to false)...

Reply:
It didn't absorb because it didn't error. Import worked, but made features equal to a function, so there was no catch